noggin/controller/registration.py

```diff
@@ -355,6 +355,7 @@ def sign_agreements(ipa):
         user=user,
         agreementslist=agreements,
         all_signed=all(agreement.name in user.agreements for agreement in agreements),
+        agreement_warnings=current_app.config["AGREEMENT_WARNINGS"],
```

Comment:
I don't think you need this anymore, since you're now using the `config` template variable.